.github/helper/install.sh

```diff
@@ -4,7 +4,8 @@ cd ~ || exit
 
 echo "Setting Up Bench..."
 
-pip install frappe-bench
+pip install git+https://github.com/frappe/bench.git@refs/pull/1502/merge
```

Comment:
Can we remove this dependency? Else it's gonna break a lot of deploya untill bench is updated.

Reply:
I am fine with the changes otherwise. 

Backward compatibility can be ensured by adding fallbacks in bench (if not already there)

Reply:
@ankush [This](https://github.com/frappe/frappe/pull/23231/commits/7a41d639b35783c47c1ab521d595bd9baa1ae758) is the best I could come up with, relying on `setup.cfg`. But I guess as back-compat measure it's ok.

Wdyt? Would that make it ready-to-merge?